2repository/build.gradle

```diff
@@ -1,5 +1,11 @@
 import com.avast.gradle.dockercompose.tasks.ComposeDown
 
+tasks.register('pushDockerImage') {
+    doLast {
+        println 'dummy pushDockerImage task'
```

Comment:
:-)

Reply:
I need a pushDockerImage task on the 2repository project to be able to call it on this project instead of every individual subproject. (I added dependencies on the subproject tasks)

Reply:
println 'Empty pushDockerImage task left here intentionally to simplify task  dependencies' ?

Reply:
> I need a pushDockerImage task on the 2repository project to be able to call it on this project instead of every individual subproject. (I added dependencies on the subproject tasks)

You can just leave the task without any action inside it, so then it does not print anything.